### DISCUSSION TITLE
Strange state of navidrome

### DISCUSSION THREAD

**Discussion by ScruffyDerHausmeister** [navidrome/navidrome#3189] [Q&A]
I've a strange Problem with Navidrome. As you see in the screenshot, the scan and quickscan buttons are greyed out and can't be clicked. In addition to that the activity button is surrounded by the progress bar. This state is now for two weeks. I've tried to restart the server (I use the docker image), I've tried to rescan the library but nothing helped. Has anyone a solution?

![image](https://github.com/user-attachments/assets/21ab3a55-5dd1-47be-aabe-a78588f1fe0a)

**Comment by deluan** [comment]
Maybe something obvious, but have you tried to refresh the page?

**Reply by ScruffyDerHausmeister** [reply]
Sure, I wouldn't ask if it was that easy. :-)

**Reply by deluan** [reply]
What happens if you load the page in a different browser, or incognito? Does it show the same status?

**Reply by ScruffyDerHausmeister** [reply]
It the same. I've tried it in Chromium, Brave and Firefox (my default browser), also tried it in FF private tab. Also on Android or desktop devices, with and without adblockers. The server was running until two weeks ago without any problems and it is still streaming! But it doesn't see any new or deleted folder or files.

**Reply by deluan** [reply]
Increase the log level to debug, restart the server and check the logs for any issues. Post the log here if you can't find any issues, as this helps troubleshooting the problem.

**Reply by ScruffyDerHausmeister** [reply]
Well, one directory could not be opened. I've gave it a+x, forced navidrone to rescan and now everything is fine... But is this right, that an directory with missing rights is blocking the whole scanning process?

**Reply by deluan** [reply]
It shouldn't. What filesystem are you using? Can you share the logs? If you don't want it public, you can send it to me, deluan @ navidrome.org

**Reply by ScruffyDerHausmeister** [reply]
Hmm, I'm using the docker image on an rasperry pi. The SSD with the music library is formatted with ext4. I don't even know if I'm saving logs or just show them in time in the console. Didn't find any logs in /data, nor in various directories in the container. I'd like to send them to you, but didn't find any... :-)

**Reply by deluan** [reply]
Use `docker logs` or `docker compose logs`

**Reply by ScruffyDerHausmeister** [reply]
I've send you some logs.
